NEW_RELEASE.md

```diff
@@ -4,10 +4,10 @@ So you think you are ready to make a release, eh? Follow the steps below :)
 
 ## Making a prelease
 - [ ] Merge main into develop
-- [ ] Make a branch off develop called VERSION-prelease
+- [ ] Make a trunk branch off develop using the version number, ie `0.3.0`
```

Comment:
Shouldn't this be a version branch? The _trunk_ branch for us is `main` unless I'm misunderstanding your proposed new workflow.

Reply:
I've gone slash & burn with this doc; have a gander at the latest version :)